deploy-web/src/pages/providers/[owner]/index.tsx

```diff
@@ -87,6 +88,33 @@ const ProviderDetailPage: React.FunctionComponent<Props> = ({ owner, _provider }
     getProviderStatus();
   };
 
+  function groupUptimeChecksByPeriod(uptimeChecks: { isOnline: boolean; checkDate: string }[] = []) {
+    const groupedSnapshots: { checkDate: Date; checks: boolean[] }[] = [];
+
+    uptimeChecks.toSorted((a, b) => new Date(a.checkDate).getTime() - new Date(b.checkDate).getTime());
```

Comment:
Didn't know about this method :) after looking into the doc and testing it seems like it's not mutating the original array. So seems like this usage might not be intended

<img width="150" alt="image" src="https://github.com/akash-network/cloudmos/assets/10720827/521ddfa4-f4dc-48a8-92b5-f03a748c2a91">

Reply:
nice, I like that too!